### PR TITLE
Moved InterpolationMethod to Canvas

### PR DESCRIPTION
**Pull request by sebastianbaginski**
Another step in adapting the raster api to canvas.
@marcusnaslund ready for review
@emilwestergren can you have a look as well

### CODE REVIEW

**Comment by marcusnaslund** [comment]
@emilwestergren is the peer reviewer for this one then.

File: source/draw/Canvas.ooc [hunk 1]
```diff
@@ -19,6 +19,11 @@ use ooc-math
 use ooc-collections
 import Image, Pen
 
+InterpolationMode: enum {
+	Fast, // nearest neighbour
```
Comment (emilwestergren) [inline source/draw/Canvas.ooc]: Comma is not necessary

Comment (sebastianbaginski) [inline source/draw/Canvas.ooc]: Enumetations in the project files are not consistent. @marcusnaslund pick one

Comment (marcusnaslund) [inline source/draw/Canvas.ooc]: With my almighty power I declare *without* to be the ooc way.

Comment (sebastianbaginski) [inline source/draw/Canvas.ooc]: ok

**Pull request merged**